hawkular-bus/pom.xml

```diff
@@ -79,28 +69,11 @@
       </dependency>
 
       <dependency>
-        <groupId>org.wildfly.core</groupId>
-        <artifactId>wildfly-controller</artifactId>
-        <version>${version.org.wildfly.core}</version>
-      </dependency>
-
-      <dependency>
         <groupId>org.wildfly</groupId>
-        <artifactId>wildfly-dist</artifactId>
+        <artifactId>wildfly-jms-client-bom</artifactId>
```

Comment:
Clicking through "usages" on http://mvnrepository.com/artifact/org.wildfly/wildfly-jms-client-bom/10.0.0.Final, it does not look like it was linked with wildfly-javaee bom. Neither I can see it anywhere when studying wildfly-jms-client-bom.